src/open_inwoner/__init__.py

```diff
@@ -1,6 +1,6 @@
 from .celery import app as celery_app  # noqa
 
-__version__ = "1.16"
+__version__ = "1.16.5"
```

Comment:
You could get the value from `RELEASE` in `open_inwoner.conf.base` (the other way around is not possible because of circular import).

Reply:
I found the circular import but didn't try the other way and that worked so I updated the PR